data/releases.yaml

```diff
@@ -1,3 +1,10 @@
+- version: v1.4.0
+  changes:
+  - add option to block exact website URL
+  - treat subdomains as separate websites when blocking sites
```

Comment:
lets add an example here

Reply:
agreed, I'll take the one I added in the PR 👍

Reply:
agreed, I'll use the examples from the PRs I made 👍

Reply:
p.s. are we meeting today?